### PR TITLE
Document 256GB SD card boot limit on all but 2837B0-based Pi's

### PR DESCRIPTION
**Pull request by andrum99**
Also remove extraneous periods from end of headings.

### CODE REVIEW

File: installation/sd-cards.md [hunk 1]
```diff
@@ -2,15 +2,17 @@
 
 The Raspberry Pi should work with any compatible SD card, although there are some guidelines that should be followed:
 
-## SD card size (capacity). 
+## SD card size (capacity) 
 
 For installation of NOOBS or the image installation of Raspbian, the minimum recommended card size is 8GB. For Raspbian Lite image installations we recommend a minimum of 4GB. Some distributions, specifically LibreELEC and Arch, can run on much smaller cards. If you're planning to use a card of 64GB or more with NOOBS, see [this page](sdxc_formatting.md) first.
 
-## SD card class. 
+**Note:** THe maximum size of SD card which most Raspberry Pis can boot from is 256GB. Only Raspberry Pi 3B+, 3A+ and Compute Module 3+ can boot from SD cards with a capacity larger than 256GB. This is due to a bug in the boot ROM within the SoC.
```
Comment (lurch) [inline installation/sd-cards.md]: THe -> The

**Comment by JamesH65** [comment]
Is this ready to go to copy edit?

File: installation/sd-cards.md [hunk 2]
```diff
@@ -2,15 +2,17 @@
 
 The Raspberry Pi should work with any compatible SD card, although there are some guidelines that should be followed:
 
-## SD card size (capacity). 
+## SD card size (capacity) 
 
 For installation of NOOBS or the image installation of Raspbian, the minimum recommended card size is 8GB. For Raspbian Lite image installations we recommend a minimum of 4GB. Some distributions, specifically LibreELEC and Arch, can run on much smaller cards. If you're planning to use a card of 64GB or more with NOOBS, see [this page](sdxc_formatting.md) first.
 
-## SD card class. 
+**Note:** The maximum size of SD card which most Raspberry Pis can boot from is 256GB. Only Raspberry Pi 3B+, 3A+ and Compute Module 3+ can boot from SD cards with a capacity larger than 256GB. This is due to a bug in the boot ROM within the SoC.
```
Comment (lurch) [inline installation/sd-cards.md]: Only a minor point, but perhaps "This is due to a bug in the boot ROM within older versions of the SoC." might be better?

**Comment by andrum99** [comment]
> Is this ready to go to copy edit?

It is now, yes.

**Comment by andrum99** [comment]
Can @ghollingworth possibly confirm that there are Pi 2's out there with the BCM2837B0 fitted? If so, how can these boards be identified? Are they a different board version, or do they only have different revision codes?

**Comment by ghollingworth** [comment]
Yeah they were Pi 2 revision 1.2, "a22042"

They are documented in https://www.raspberrypi.org/documentation/hardware/raspberrypi/revision-codes/README.md

**Comment by andrum99** [comment]
Thanks @ghollingworth. Am I right in saying all Pi 2s that are being produced have the BCM2837B0?

**Comment by lurch** [comment]
@ghollingworth In case it's not clear, I think @andrum99 is specifically trying to clarify the situation between Pi2s with BCM2837A0 (plastic case) and BCM2837B0 (metal case) ?

**Comment by ghollingworth** [comment]
No there are no Pi 2s with B0 fitted and no plans to do this

**Comment by andrum99** [comment]
> No there are no Pi 2s with B0 fitted and no plans to do this

That's not what you said above @ghollingworth . So just to confirm, there has never been a Pi 2 with a B0 variant on it?

**Comment by ghollingworth** [comment]
Everything I've said above is true, there were some Pi 2 r1.2 which had a revision "a22042", but none with B0 silicon.

**Comment by andrum99** [comment]
@ghollingworth I asked you specifically about revision B0 and you said yes - there were some Pi 2s with that variant. I'm happy we've got the situation clarified now so that's fine.

Secondly, Can you confirm that there are some Pi 3Bs (not plus) that have B0 silicon, and also whether this is true of all Pi 3Bs (not plus) that are being manufactured?

**Comment by helenlynn** [comment]
Copy checked, no edits necessary.

File: installation/sd-cards.md [hunk 3]
```diff
@@ -2,15 +2,17 @@
...
+**Note:** Only the Raspberry Pi 3A+, 3B+ and Compute Module 3+ can boot from an SD card larger than 256 GB. This is because there was a bug in the SoC used on previous models of Pi. 
+
+## SD card class
 
 The card class determines the sustained write speed for the card; a class 4 card will be able to write at 4MB/s, whereas a class 10 should be able to attain 10 MB/s. However, it should be noted that this does not mean a class 10 card will outperform a class 4 card for general usage, because often this write speed is achieved at the cost of read speed and increased seek times.
 
-## SD card physical size. 
+## SD card physical size
 
 The original Raspberry Pi Model A and Raspberry Pi Model B require full-size SD cards. The newer [Raspberry Pi Model A+](https://www.raspberrypi.org/products/raspberry-pi-1-model/), [Raspberry Pi Model B+](https://www.raspberrypi.org/products/raspberry-pi-1-model-b/), [Raspberry Pi 2 Model B](https://www.raspberrypi.org/products/raspberry-pi-2-model-b/), [Raspberry Pi Zero](https://www.raspberrypi.org/products/raspberry-pi-zero/), and [Raspberry Pi 3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/) require micro SD cards.
```
Comment (lurch) [inline installation/sd-cards.md]: The A+ and B+ links need updating, it seems they're now at
https://www.raspberrypi.org/products/raspberry-pi-1-model-a-plus/
https://www.raspberrypi.org/products/raspberry-pi-1-model-b-plus/

Comment (lurch) [reply installation/sd-cards.md]: ...although I guess this last sentence _could_ be replaced with "All newer models of Raspberry Pi require micro SD cards." :grey_question:

Comment (JamesH65) [reply installation/sd-cards.md]: Done.

**Pull request merged**